README.md

```diff
@@ -36,10 +36,18 @@ The application is a lottery for an e-commerce site. This is an open source appl
 ## Goals & constraints
 
 ### Goals
-As a company, the idea is to leverage an open source cloud native application to use it internally. Because the application is a cloud native one and will have to be frequently improved, the choice is to use the devops methodology to maintain this application into production.
+You are in a company which achieved an aggressive deal with a Customer to leverage an open source cloud native application and use it internally.
+
+You as a new team are in charge to deliver and maintain the application in production in your customer environment.
+The custoner gave you 3 major objectives:
+ - The application is an open source cloud native application
+ - The customer cloud production environment must be Openstack
+ - Any feature/bug request must be in production quickly, with quality.
+
+To answer to customer need, your team choose to use the DevOps methodology to deploy and maintain this application into production in the customer company.
```

Comment:
needs

Is DevOps a methodology really ? Rather an "approach" to join developers to operators forces to satisfy customer requests no ? V-Cycle or agile are more methodologies.

in production.